fixed-hash/src/hash.rs

```diff
@@ -332,7 +332,7 @@ macro_rules! impl_heapsize_for_hash {
 	}
 }
 
-#[cfg(not(feature="heapsizeof"))]
+#[cfg(any(not(feature="heapsizeof"), not(feature="libc"), target_os = "unknown"))]
```

Comment:
Probably `not(<other-condition>)` would make it a bit easier to maintain in the future, but still ok for me.

Reply:
@niklasad1 merge at will